File: solutions/c/02-cz2/code/app/main.c
#include <stdio.h>
#include <string.h>

int main() {
  // Flush after every printf
  setbuf(stdout, NULL);

  printf("$ ");

  // Wait for user input
  char input[100];
  fgets(input, 100, stdin);
  // Remove the trailing newline
  input[strlen(input) - 1] = '\0';
  printf("%s: command not found\n", input);

  return 0;
}

File: solutions/php/02-cz2/code/app/main.php
<?php
error_reporting(E_ALL);

fwrite(STDOUT, "$ ");

// Wait for user input
$command = fgets(STDIN);
if ($command === false) {
    $command = '';
}
$command = rtrim($command, "\r\n");
fwrite(STDOUT, $command . ": command not found\n");
